src/Main.java

```diff
@@ -15,32 +15,33 @@ public static void main(String[] args) throws Exception {
     System.out.println("You have " + letsPlay.getGuessesLeft() + " guesses left");
     System.out.println(letsPlay.getRevealedMovieTitle());
 
-    while (letsPlay.isGameInPlay()) {
+    while (!letsPlay.isHasWon() && !letsPlay.isLastGuess()) {
```

Comment:
I think I liked this on more before, 
``` 
while (letsPlay.gameInProgress()){
...
```

gameInProgress would then do the check isHasOne and isLastGuess internally.